packages/contract/src/DaimoPaymaster.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.12;
+
+import "account-abstraction/core/BasePaymaster.sol";
+
+import "./DaimoAccount.sol";
+
+contract DaimoPaymaster is BasePaymaster {
+    mapping(address => bool) public destWhitelist;
+    mapping(address => bool) public senderBlacklist;
```

Comment:
I'd put a short comment above saying
- Purpose (free ops on Daimo, especially meta-ops like Add Device)
- In what conditions DaimoPaymaster will sponsor a transaction
- Exceptions when it won't

Reply:
done